src/main/java/no/nav/veilarbperson/controller/PersonController.java

```diff
@@ -77,14 +77,7 @@ public HarLoggetInnRespons harNivaa4(@PathVariable("fodselsnummer") Fnr fodselsn
         authService.stoppHvisEksternBruker();
         authService.sjekkLesetilgang(fodselsnummer);
 
-        if (unleashService.isEnabled("veilarb.sjekk.nivaa4")) {
```

Comment:
Denne kan vel ikke slettes? da vil man aldri kunne teste i testmiljø? Evt, kunne vi hatt en egen toggel for både nivå 4 + id porten som alltid er på i test så man slipper å ordne med testbrukere hele tiden?